_studio/mfx_lib/cmrt_cross_platform/src/cmrt_cross_platform.cpp

```diff
@@ -204,13 +202,7 @@ INT CreateCmDevice(CmDevice *& pD, UINT & version, VADisplay va_dpy, UINT mode)
     CmDeviceImpl * device = new CmDeviceImpl;
 
     device->m_platform = VAAPI;
-#if defined(ANDROID)
-    device->m_dll = vm_so_load(DLL_NAME_ANDROID);
-    if (device->m_dll == 0)
-        device->m_dll = vm_so_load(DLL_NAME_LINUX);
-#else
     device->m_dll = vm_so_load(DLL_NAME_LINUX);
```

Comment:
Why this went away? @dmitrybrazhkin : FYI

Reply:
The same.